### DISCUSSION TITLE
ダウンロード出来ない

### DISCUSSION THREAD

**Discussion by akiran7607** [aclap-dev/video-downloadhelper#1524] [Help and Q&A]
### Your question

https://www.udemy.com/course/
からの動画がダウンロード出来ません。
ライセンス購入済です。
対応頂けることを楽しみにしております。
宜しくお願い致します。
------------------------------------------------
I am unable to download videos from https://www.udemy.com/course/.
I have already purchased the license.
I look forward to your support on this matter.
Thank you in advance.

### Addon & CoApp version and details

![無題](https://github.com/user-attachments/assets/ca4fc4db-1a16-4172-9075-fd0856ffde46)

**Comment by Mattmjs** [comment]
Which browser are you using ? Firefox will show you if video is protected by DRM
https://github.com/aclap-dev/video-downloadhelper/discussions/1306#discussioncomment-10786127

**Reply by akiran7607** [reply]
I DON'T USE FIREFOX.
I USE CHROME.
I BOUGHT CHROME LICENSE.

**Reply by Mattmjs** [reply]
That's fine but if you have a look in Firefox, take a full screenshot of the Firefox window and attach the image here.
It might show it's protected.
